### PR TITLE
Maintenance release 1

### PR DESCRIPTION
**Pull request by weralabaj**
### Improvements/Features
- [__#258__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/210) Support Newtonsoft.Json 11
- [__#98__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/98) Implement pessimistic saga concurrency
- [__#210__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/210) Add Outbox config options similar to other persistences
- [__#276__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/276) Allow SQL persistence to use different connection strings for subscriptions and timeouts

### Bug fixes
- [__#270__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/270) SqlPersistenceSession throws in endpoint without sagas/outbox
- [__#272__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/272) Install actions fail silently when running via NSB host
- [__#260__](https://github.com/Particular/NServiceBus.Persistence.Sql/issues/260) Concurrent connections in same TransactionScope throw Exceptions

### CODE REVIEW

**Comment by weralabaj** [comment]
rebased on top of develop

File: src/SqlPersistence/Outbox/SqlPersistenceOutboxSettingsExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+            configuration.GetSettings().Set(SqlOutboxFeature.FrequencyToRunDeduplicationDataCleanup, frequencyToRunDeduplicationDataCleanup);
+        }
+
+        /// <summary>
+        /// Disable the built-in outbox deduplication records cleanup.
+        /// </summary>
+        public static void DisableCleanup(this OutboxSettings configuration)
+        {
+            configuration.GetSettings().Set(SqlOutboxFeature.DisableCleanup, true);
```
Comment (SimonCropp) [inline src/SqlPersistence/Outbox/SqlPersistenceOutboxSettingsExtensions.cs]: missing guard

Comment (tmasternak) [reply src/SqlPersistence/Outbox/SqlPersistenceOutboxSettingsExtensions.cs]: Missed your comment. Will fix on the release branch. Thx!

Comment (tmasternak) [reply src/SqlPersistence/Outbox/SqlPersistenceOutboxSettingsExtensions.cs]: Add in https://github.com/Particular/NServiceBus.Persistence.Sql/pull/294

**Pull request merged**